### ISSUE TITLE
Warn about `col-xx-0`?

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootlint#166]
From the Bootstrap CDN src...

``` html
<div class="col-sm-0 col-md-2">
  <div class="tryit">
    <a href="javascript:tryIt(1)" class="btn btn-primary btn">Try it!</a>
  </div>
</div>
```

Should we warn about those?
